index.html

```diff
@@ -922,9 +922,11 @@
 
 A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define
 them. Their identity *in a [=context=]* is the set of characteristics they
-present in that context. People frequently present different identities to
-different contexts, and also frequently share an identity among several
-contexts.
+present to that context, or that the context can infer.  This set of
```

Comment:
```suggestion
present to that [=context=], or that they can reasonably expect observing [=parties=] to infer from their behavior in that [=context=].  This set of
```

Reply:
I like what you're saying, but:
* I don't think that contexts can infer anything (a party observing you in the context can).
* I don't want to say that any inference is potentially your identity. Maybe being gay is statistically related to, I don't know, the enjoyment of cheese.<sup>1</sup> If you're not out when you eat at my restaurant, I shouldn't consider that your orientation is part of your identity in that context even if it's pretty clear from your eating history.

<sup>1.</sup> For a time I worked on a small research team in which I was the only nongay person and the only one who didn't hate cheese. Result for small `n`, though.

Reply:
I get your point, but I think this goes against the way websites are actually abusing identity on the web today which is why I aimed for the direction I did, but we've not really landed anywhere close to it. There's numerous sites which infer attributes based on behavioral usage or specific device information which aren't reasonably expected by the user. Take for example, [this story](https://www.forbes.com/sites/kashmirhill/2012/02/16/how-target-figured-out-a-teen-girl-was-pregnant-before-her-father-did/?sh=4df7391a6668) about Target. Reasonably, no one expected Target to be able to infer this information based on purchasing behavior. And this is data that the customer could expect Target to know and collect. If they had this wouldn't be a news story. Similarly people wouldn't be constantly talking about how their microphone is listening to them when in reality that's just how good of inferential data can be derived today and no one reasonably understands it. Except this is actually what happens.

This is why I was aiming for a direction of making it clear that the browser should be acting in the users best interest in creating a default profile to remove the ability to infer as much as possible. It's become clear that's not possible as it would break webcompat in serious ways because lots of these correctable attributes have become useful to use and that's understandable. However, looking at where I started from versus where this language has ended up puts into perspective how far apart we are from where I believe many want to go (and describe principles of) versus what we can realistically state. This is what I meant by we'd wordsmithed all the meaning out of this.